int8 sensorID
float32 value
